### DISCUSSION TITLE
Why I am seeing this?

### DISCUSSION THREAD

**Discussion by 0xParth** [projectdiscovery/httpx#1703] [General]
<img width="1474" alt="image" src="https://github.com/projectdiscovery/httpx/assets/86850255/cea48b2d-f5c1-4b1b-9267-746e105ffc9a">

why I am seeing this?? I removed everything, rebuilt from scratch still.

**Comment by GeorginaReeder** [comment]
Just noting that we've answered this question in Discord, here: https://discord.com/channels/695645237418131507/1240361026898235442/1240361026898235442

As mentioned, that's a python library that comes with some distributions of linux. We have a video that should help with fixing that: https://nux.gg/wrong-httpx

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
